pom.xml

```diff
@@ -621,6 +622,9 @@
           <shutdown>exit</shutdown>
           <trimStackTrace>false</trimStackTrace>
           <forkNode implementation="org.apache.maven.plugin.surefire.extensions.SurefireForkNodeFactory"/>
+          <systemPropertyVariables>
```

Comment:
please add the same to the common configuration section of the failsafe plugin.
Hbase doesn't use failsafe, but Phoenix does.